arkouda/pdarrayclass.py

```diff
@@ -65,6 +65,30 @@ def unescape(s):
         raise ValueError(("unsupported value from server {} {}".\
                               format(mydtype.name, value)))
 
+
+
+
+@typechecked
+def parse_single_int_array_value(msg : str) -> object:
```

Comment:
If this is to be called internally (within this module only), I recommend prefacing the method name with an underscore

Reply:
good! it has been changed as  _parse_single_int_array_value